pkg/omf/omf.fish

```diff
@@ -136,11 +138,14 @@ function omf -d "Oh My Fish"
 
     case "u" "update"
       echo (omf::em)"Updating Oh My Fish..."(omf::off)
-      if omf.repo.pull $OMF_PATH
-        echo (omf::em)"Oh My Fish is up to date."(omf::off)
-      else
-        echo (omf::err)"Oh My Fish failed to update."(omf::off)
-        echo "Please open a new issue here → "(omf::em)"github.com/oh-my-fish/oh-my-fish/issues"(omf::off)
+      switch omf.repo.pull $OMF_PATH
```

Comment:
Yeah. This `switch` doesn't work. But just to get the idea here: We need a way to know if `omf.repo.pull` have updated or not the target repository. This is needed to avoid refreshing when nothing gets updated.